### DISCUSSION TITLE
RegisterResourceHandler

### DISCUSSION THREAD

**Discussion by dtamade** [cefsharp/CefSharp#3363] [General]
RegisterResourceHandler 
Is it stable??

Why do my multiple registerresourcehandlers fail to work

And re

**Comment by amaitland** [comment]
To better understand what's going on I'd suggest you look at the source.

https://github.com/cefsharp/CefSharp/blob/8a367fbdc4035c149dc657b37021a91753777c37/CefSharp/WebBrowserExtensions.cs#L463
https://github.com/cefsharp/CefSharp/blob/13e74fea8dcda826d602d1ee534b67f46e93bd15/CefSharp/ResourceRequestHandlerFactory.cs#L91

You can implement IResourceRequestHandlerFactory and provide a custom implementation if you a specific use case.

**Reply by amaitland** [reply]
If you require further assistance then please answer the questions listed at https://github.com/cefsharp/CefSharp/wiki/Questions-&-Support#when-asking-a-question-please-include-the-following
